### ISSUE TITLE
Catégories non sélectionnées par défaut

### ISSUE THREAD

**Issue by sebastienaubry** [2p2r/velobs_web#286]
Quand on arrive sur https://velobs.2p2r.org/ , par défaut, aucune catégorie "feuille" n'est sélectionnée :
![image](https://user-images.githubusercontent.com/41120158/235323370-a6fefde9-09de-40aa-8f16-b47b96c397eb.png)

Constaté en navigation privée sous Firefox sur plusieurs ordinateurs.

**Comment by gerald2545** [comment]
OK, je n'avais pas bien compris ton problème, je confirme.
tu confirmes qu'il se souvient de ta sélection quand tu reviens sur le site web, après fermeture de l'onglet et ou/ navigateur

**Comment by gerald2545** [comment]
@sebastienaubry 
petite question en attente ;)

**Comment by sebastienaubry** [comment]
Non, il s'agit de la sélection par défaut pour un nouvel utilisateur.
Il suffit d'ouvrir une fenêtre de navigation privée et d'aller sur https://velobs.2p2r.org/ pour le constater.
